### ISSUE TITLE
Overmind Mode: 1 Thread commands all bot from Spectrator Seat

### ISSUE THREAD

**Issue by davidmann4** [davidmann4/ogar-feeder-bot#201]
- add the mode to config.js
- add the new behavior to the feeder.js
- emit a packet when a spectrator bot has new commands for the feeder bots
- distribute the different movment commands via the server (direct message)

**Comment by hassanila** [comment]
@davidmann4 That's great and it would easily work but how will the spectating bot be tracking you?

**Comment by davidmann4** [comment]
every ball in the game has a unique identifier (ball_id) it stays the same through different connections

**Comment by hassanila97** [comment]
@davidmann4 Alright hope someone makes a pull request but I think many don't want to mention Agar.io as you say this is only for Ogar...

**Comment by WJKH** [comment]
Right, I don't understand why @davidmann4 change the name of the project and try to remove Agar.io from this project, is there anything we don't know?

Everyone here trying to help in any way is coming because they want Agar.io bots.

**Comment by thisismo** [comment]
agar.io takes legal action against botters and you are not allowed to use 'agar' because it's copyrighted

**Comment by WJKH** [comment]
Ok, I didn't know that.

**Comment by MasonBurdette** [comment]
Agar is a science related substance.  Why can I not use that name?  I thought a generic name couldn't be trademarked/copyrighted.

**Comment by hassanila97** [comment]
@MasonBurdette @WJKH Because now they've made it as business so if you use it the "kill" you. I don't care and I will use Agar.io and Agar as much as I want, it is a public name....

**Comment by picusc** [comment]
Maybe using Agar is possible not Agar.io

**Comment by hassanila97** [comment]
Ok then instead of Ogar I want to use Agar..  Thanks

**Comment by WJKH** [comment]
If Agar want stop bots, first they should ban bot sellers IPs as 90% of bots come from those websites.
Simple and easy.
